### PR TITLE
edited src by the instructions

### PR DESCRIPTION
**Pull request by itasgaming**

### CODE REVIEW

File: src/Teleop.elm [hunk 1]
```diff
@@ -109,9 +109,8 @@ view model =
                 [ createButton ColorRoulette "spun to\ncorrect color?"
                 , printButton "no" "yes" model.colorRoulette
                 ]
-            , column
-                yophyTophy
-                [ createButton SpinedRoulette "spun 3-5?"
+            , column yophyTophy
+                [ createButton SpinedRoulette "spun cycles 3-5?"
```
Comment (HadarNahshon) [inline src/Teleop.elm]: Would be more correct as-
> spun 3-5 cycles?

plus- `SpinedRoulette` should be either `SpinnedRoulette` or `SpunRoulette`.

**Comment by HadarNahshon** [comment]
Nice work overall, but you should've requested a pull request for each issue separately.

**Review by HadarNahshon — APPROVED** [review]
I was a bit tired, so may have missed some stuff.

File: src/TeamData.elm [hunk 2]
```diff
@@ -36,7 +46,22 @@ view model =
...
         ]
         [ textInput model.scouterName ScouterInput "Scouter's name"
-        , textInput model.station StationInput "Scouted station"
+        , radioRow
+            [ padding 10
+            , spacing 20
+            ]
+            { onChange = Station
+            , selected = Just model.station
+            , label = Input.labelAbove [] (text "Witch station?:")
```
Comment (HadarNahshon) [inline src/TeamData.elm]: ** Which

**Review by tombavarsky — APPROVED** [review]

File: src/TeamData.elm (hunk 2 above)
Comment (HadarNahshon) [reply src/TeamData.elm]: And no need for a colon after the question mark.

**Review by HadarNahshon — APPROVED** [review]

**Comment by HadarNahshon** [comment]
I think that a more indicative name would be better, but nicely done.

**Pull request merged**